rkt/image/filefetcher.go

```diff
@@ -85,13 +89,21 @@ func (f *fileFetcher) getVerifiedFile(aciPath string, a *asc) (*os.File, error)
 	if err != nil {
 		return nil, errwrap.Wrap(errors.New("error opening signature file"), err)
 	}
-	defer func() { maybeClose(ascFile) }()
+	defer func() {
+		if ascFile != nil {
+			ascFile.Close()
+		}
+	}()
```

Comment:
since [`*asc.Get`](https://github.com/coreos/rkt/pull/3120/files#diff-bbca85443e1d6dedcff42f176ad118a9R84) now returns in case of non-err a closable `readSeekCloser` just `defer ascFile.Close()`.